NumberValidator/Helpers/LuhnAlgorithm.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace NumberValidator.Helpers
+{
+    public class Luhn
+    {
+        public static void Validate(string input)
+        {
+            int sum = 0;
```

Comment:
in C# we use `var` instead of explicitly specifying type. types are inferred automatically

Reply:
ok i will change it